pages/acknowledgements.js

```diff
@@ -2,14 +2,71 @@ import React from "react";
 
 import SharedLayout from "components/SharedLayout";
 import BackButton from "components/navigation/BackButton";
-import PlaceholderText from "components/mock/Placeholder";
 
 export default function Acknowledgements() {
     return (
         <SharedLayout>
             <BackButton className="mt-3 mb-2" />
             <h1>Acknowledgements</h1>

```

Comment:
There's this rather obscure h1 rule:

> - Use only one `<h1>` per page or view. It should concisely describe the overall purpose of the content.
> - Using more than one `<h1>` will not result in an error, but is not considered a best practice. It is beneficial for screenreader users, and SEO.
> - While HTML5 allows a `<h1>` per sectioning element, it is not considered best practice, and may subvert the expectations of how screen reader users navigate.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes

If time allows, it would be good to change this to an h2 and style the element appropriately
```suggestion
            <h2>Acknowledgements</h2>
```